tests/randomized/README.md

```diff
@@ -156,6 +156,46 @@ Then load it in `gdb`:
 gdb --core=/tmp/core php-fpm|httpd|php
 ```
 
+### Analyzing using ASAN
+
+In some cases it might be tricky to find the memory corruption, but you can
+always use an address sanitizer to try and find problems.
```

Comment:
Could you please add one sentence that the installer needs be be fetched from the `package extension zts-debug-asan` step instead of `package extension` if the tracer itself should also be linked against asan?